transform/mattermost-analytics/models/staging/cws/_cws__models.yml

```diff
@@ -64,3 +64,23 @@ models:
       - name: is_terms_accepted
         description: Whether the user accepted the terms and conditions.
 
+  - name: stg_cws__marketing
+    description: |
+      CWS Marketing data.
+
+    columns:
+      - name: marketing_id
+        description: Unique Identifier for Marketing data.
+        tests:
+          - not_null
+      - name: email
```

Comment:
Tests need to be added in order to ensure valid input values for this field.

Reply:
Should we use validate_email macro in the model or `not_null` and `dbt_utils.not_empty_string` would be sufficient?

Reply:
`validate_email` doesn't check for nulls. Empty string will raise an error for missing `@`. Using a combination of expression with `validate_email` and `not_null` should be enough.

Reply:
Added `validate_email` to the model

Reply:
`not_null` and `empty_string` were added to the yaml file

Reply:
Resolving in favor of the other comment for `is_valid_email`